### PR TITLE
Adding Steps.tsx in Maintainers folder

### PR DESCRIPTION
**Pull request by develad**
localhost_3000_maintainers(Nest Hub Max)
localhost_3000_maintainers(Nest Hub Max) (1)
localhost_3000_maintainers(Nest Hub Max) (2)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,18 +17,24 @@
     ]
   },
   "dependencies": {
+    "@radix-ui/react-accordion": "^1.1.2",
     "autoprefixer": "10.4.14",
+    "class-variance-authority": "^0.7.0",
     "classnames": "^2.3.2",
+    "clsx": "^2.0.0",
```
Comment (adirkandel) [inline package.json:24]: We already use `classnames` to generate dynamic classNames

Comment (develad) [reply package.json:24]: @adirkandel 
I've coded the whole thing (maintainers page) in one day and hadn't took up the time to check the current packages state.
Can you make the proper adjustments that we can omit this package?

**Comment by develad** [comment]
@adirkandel 
Clsx package removed. Awaiting review 🕙
https://github.com/Maakaf/maakaf-website/pull/96#discussion_r1323973765

File: package.json (hunk 1 above)
Comment (develad) [reply package.json]: Issue resolved in https://github.com/Maakaf/maakaf-website/pull/96/commits/3f6e3c5b29f58b9f8bb481a633a5dee04c2ab849 commit

**Comment by adirkandel** [comment]
I see now that you also implemented the QnA accordion here (which is out of scope of this PR) and also in
https://github.com/Maakaf/maakaf-website/pull/93

It's hard to review it that way, can you make this PR more concise?

**Comment by develad** [comment]
@adirkandel 
> @develad You did an amazing job buddy, tnx!
> 
Thank you!

https://github.com/Maakaf/maakaf-website/pull/96#issuecomment-1717835459
I arranged each PR with its corresponding files.. (MaintainerForm #91 QuestionsAnswers #93 Steps #96 ) so it would be easy for the maintainers to review  
https://github.com/Maakaf/maakaf-website/pull/93#issuecomment-1717813194
> I see that you installed radix accordion so, did you use radix or shadcn?
> 
https://ui.shadcn.com/docs
radixUI

**Review by adirkandel — APPROVED** [review]

**Pull request merged**